src/js/stores/AppFormStore.js

```diff
@@ -295,16 +295,15 @@ function processResponseErrors(responseErrors, response, statusCode) {
       AppFormErrorMessages.getGeneralMessage("unknownServerError");
 
   } else if (statusCode === 422 && response != null &&
-      Util.isArray(response.errors)) {
-
-    response.errors.forEach((error) => {
-      var attributePath = error.attribute.length
-        ? error.attribute
+      Util.isArray(response.details)) {
```

Comment:
Question:
Is the `response.errors` gone completely or wasn't it there at any time?
Shouldn't we handle both `response.errors` and `response.details`?
I think there is an inconsistency from the service layer. Sometimes that, sometimes this.
But I am not sure.